src/datastore/postgres-store.ts

```diff
@@ -2936,6 +2937,42 @@ export class PgDataStore extends (EventEmitter as { new (): DataStoreEventEmitte
     });
   }
 
+  async getAddressNFTEvent(args: {
+    stxAddress: string;
+    limit: number;
+    offset: number;
+  }): Promise<{ results: AddressNftEventIdentifier[]; total: number }> {
+    return this.query(async client => {
+      const result = await client.query<AddressNftEventIdentifier & { count: string }>(
+        `
+        WITH lastTransfers AS (
+          SELECT MAX(block_height) as max_block_height, asset_identifier, value
+          FROM nft_events
+          WHERE canonical = true AND (sender = $1 OR recipient = $1)
```

Comment:
What is the purpose of matching against `sender` here?

Reply:
to incorporate the `Burn` type of NFTs. shouldn't it?

Reply:
You're right It should only check against the recipient.

Reply:
does this check make sense for `Transfer` type NFTs?